### ISSUE TITLE
I made a game for this

### ISSUE THREAD

**Issue by liamoc** [tebl/RC6502-Apple-1-Replica#25]
This isn't really an issue, but I thought it may be of interest to people who follow this project.

Previously I made a minesweeper clone in 6502 assembly for the Apple II, BBC Micro, Commodore 64 and Atari 8-bit line. It has music and sound on the latter three too.

Anyway, today I ported it to the RC6502 with a color VDU and the joystick controller addons. It doesn't have sound effects but does show LED animations sometimes.

The (messy) source code for all versions is available at https://github.com/liamoc/6502_mep and I've attached the requisite wozmon input file to play it straight away. 

[main.hex.txt](https://github.com/tebl/RC6502-Apple-1-Replica/files/6970558/main.hex.txt)

Also attached is a screenshot (apologies for the poor quality), just to show the game running:

![IMG_3161](https://user-images.githubusercontent.com/61991/129082461-88b40a37-096d-4e23-93c0-a0cf7adb236c.jpg)

Edit: It would be trivial to reassemble this to run as a ROM image, btw, because it doesn't depend on any of the existing ROM code.  It would just mean changing the location in the assembler configuration to be ROM rather than RAM.

**Comment by tebl** [comment]
That is cool! Thank you for sharing 😀

ons. 11. aug. 2021, 20:21 skrev liamoc ***@***.***>:

**Comment by 0cjs** [comment]
Yes, this is great work! When contemplating building things like the video board, it's very motivating to have some demo software immedaitely available.

Here's a thought to consider: it might be worth adding a new documentation file (`SOFTWARE.md`?) linking to software like this, specifically designed for this system. (And maybe more generic software, too, if folks feel the need).

A clean way to do this would be for you to [create a release][rel] for your GitHub project containnig the Wozmon file, and then the file here could link to [your project release page][proj]. (Doing releases is more reliable than committing object files to the repo and trying to keep them in sync.) The creation of releases can also be automated to some degree.

[rel]: https://docs.github.com/en/github/administering-a-repository/releasing-projects-on-github/managing-releases-in-a-repository
[proj]: https://github.com/liamoc/6502_mep/releases

**Comment by liamoc** [comment]
Done!

https://github.com/liamoc/6502_mep/releases/tag/v1.0
